service-deployment/bootstrap/hadoop-service/hadoop-data-node.yaml.template

```diff
@@ -40,6 +40,13 @@ spec:
           name: hadoop-config-volume
         - mountPath: /host-configuration
           name: host-confg-volume
+        readinessProbe:
+          exec:
+            command:
+            - cat
+            - /jobstatus/jobok
```

Comment:
>/jobstatus/jobok [](start = 14, length = 16)

is this file in host filesystem? if yes, what if datanode node manager collocated together?